pkg/restapi/webhook/payloads.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package webhook
+
+// ConnectionMsg is sent when a pairwise connection record is updated.
```

Comment:
Add a TODO saying these models are not final, you just copied models from arcpy, we may not need all of these.

Reply:
Done

Reply:
Also associated the TODOs with stories.

Reply:
Also added a test for this.